concepticondata/conceptlists/Bowern-2012-348.tsv

```diff
@@ -0,0 +1,349 @@
+NUMBER	ID	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	COMMENTS
+1	Bowern-2012-348-1	1dl.excl	2636	WE TWO (EXCLUSIVE)	
+2	Bowern-2012-348-2	1dl.incl	2637	WE TWO (INCLUSIVE)	
+3	Bowern-2012-348-3	3dl	2635	THEY TWO	
+4	Bowern-2012-348-4	Bloodwood	3007	EUCALYPT	
+5	Bowern-2012-348-5	Blue-Tongue Lizard	632	LIZARD	mapping to LIZARD as it's the only Lacertilia in the list
+6	Bowern-2012-348-6	Brolga	1830	CRANE	mapping to CRANE as it's the only Gruidae in the list
+7	Bowern-2012-348-7	Carpet snake	1563	PYTHON	mapping to PYTHON as it's the only Pythonidae in the list
+8	Bowern-2012-348-8	Crow	913	CROW	
+9	Bowern-2012-348-9	Death adder	3008	DEATH ADDER	
+10	Bowern-2012-348-10	Duck	1360	DUCK	
+11	Bowern-2012-348-11	Echidna	3009	ECHIDNA	
+12	Bowern-2012-348-12	Emu	2817	EMU	
+13	Bowern-2012-348-13	I	1209	I	
+14	Bowern-2012-348-14	Magpie	2039	MAGPIE	
+15	Bowern-2012-348-15	Nest	539	NEST	
+16	Bowern-2012-348-16	Possum	470	POSSUM	
+17	Bowern-2012-348-17	Scorpion	1538	SCORPION	
+18	Bowern-2012-348-18	Shield	1064	SHIELD	
+19	Bowern-2012-348-19	Sulphur-Crested Cockatoo	3010	WHITE COCKATOO	
+20	Bowern-2012-348-20	Willy Wagtail	3011	WAGTAIL	
+21	Bowern-2012-348-21	Witchetty Grub	3012	GRUB	
+22	Bowern-2012-348-22	above	1741	ABOVE	
+23	Bowern-2012-348-23	again	204	AGAIN	
+24	Bowern-2012-348-24	all	98	ALL	
+25	Bowern-2012-348-25	alone	1964	ALONE	
+26	Bowern-2012-348-26	and	1577	AND	
+27	Bowern-2012-348-27	angry	1826	ANGRY	check if it shouldn't be BE ANGRY
+28	Bowern-2012-348-28	ankle	803	ANKLE	
+29	Bowern-2012-348-29	ant	587	ANT	
+30	Bowern-2012-348-30	armpit	1886	ARMPIT	
+31	Bowern-2012-348-31	ash	646	ASH	
+32	Bowern-2012-348-32	ask	1309	ASK (INQUIRE)	check if it should be ASK (REQUEST)
```

Comment:
ask-request is normally not used, or it is made explicit, so ask inquire is good!